rrfs-test/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+# (C) Copyright 2017-2023 UCAR
```

Comment:
we are not UCAR, just remove this whole copyright

Reply:
@CoryMartin-NOAA Thanks for pointing this out. Any template on this for EMC ( or an appropriate one) copyrights?

Reply:
No copyright needed, if we have a single license at the top level, that's more than sufficient. Plus fed employees technically can't copyright things

Reply:
I changed to the current words.